### PR TITLE
[build]: Provide support for apt install packages

### PR DESCRIPTION
**Pull request by VenkatCisco**
#### Why I did it
apt-package handling: These are part of the Export variables for .j2 files and is needed for Debian and its derivatives.  

#### How I did it
Add support to slave.mk files to export APT_PACKAGES and DBG_APT_PACKAGES

#### How to verify it

The apt package, provides the apt management tool, a high-level command-line interface for better interactive usage. APT also includes command-line programs for dealing with packages, which use the library. Three such programs are apt, apt-get and apt-cache and can be verified for their existence. 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: slave.mk [hunk 1]
```diff
@@ -741,6 +741,8 @@ $(addprefix $(TARGET_PATH)/, $(DOCKER_IMAGES)) : $(TARGET_PATH)/%.gz : .platform
 		$(eval export $(subst -,_,$(notdir $($*.gz_PATH)))_pydebs=$(shell printf "$(subst $(SPACE),\n,$(call expand,$($*.gz_PYTHON_DEBS)))\n" | awk '!a[$$0]++'))
 		$(eval export $(subst -,_,$(notdir $($*.gz_PATH)))_whls=$(shell printf "$(subst $(SPACE),\n,$(call expand,$($*.gz_PYTHON_WHEELS)))\n" | awk '!a[$$0]++'))
 		$(eval export $(subst -,_,$(notdir $($*.gz_PATH)))_dbgs=$(shell printf "$(subst $(SPACE),\n,$(call expand,$($*.gz_DBG_PACKAGES)))\n" | awk '!a[$$0]++'))
+		$(eval export $(subst -,_,$(notdir $($*.gz_PATH)))_dbgs=$(shell printf "$(subst $(SPACE),\n,$(call expand,$($*.gz_DBG_PACKAGES)))\n" | awk '!a[$$0]++'))
```
Comment (lguohan) [inline slave.mk:744]: is this duplicated?

**Comment by lguohan** [comment]
@xumia, can you take a looK?

**Comment by xumia** [comment]
> @xumia, can you take a looK?

The PR looks good to me.
It has version small impact for reproducible build, when a version not specified in version control file, we will use the latest version. I have created an issue https://github.com/Azure/sonic-buildimage/issues/7363 to fix it. Our auto-upgrade version azure pipeline can detect all the debian packages, not relative to how a package is installed.

**Review by xumia — APPROVED** [review]

**Comment by lguohan** [comment]
@venkatCisco, do we need to backport these to released branch?

**Comment by VenkatCisco** [comment]
> @VenkatCisco, do we need to backport these to released branch? looks like there is more work to be done on w.r.t reproducible build part.

I see @xumia raised PR#7363 to address reproducible builds. We can park this in sonic/master until that resolves ?

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**